Guidelines.md

```diff
@@ -1577,7 +1577,7 @@ resource        | Yes      | Resource path to watch.
 notificationUrl | Yes      | The target web hook URL.
 clientState     | No       | Opaque string passed back to the client on all notifications. Callers may choose to use this to provide tagging mechanisms.
 
-If the subscription was successfully created, the service MUST respond with the status code 201 CREATED and a body containing at least the following properties, in the same order as the request array, of the subscription:
+If the subscription was successfully created, the service MUST respond with the status code 201 CREATED and a body containing at least the following properties:
```

Comment:
The way this sentence is written is a little confusing. At first, it sounds like it's talking about the order of the properties on the response object. As @garethj-msft pointed out, the document previously states that clients should not depend on the order of properties. That would be enough reason to remove it. However, when reading the sentence again I also notice that it says:

> the same order as the /request array/

This is weird because the request is not an array, and nothing in the request is an array. The subscription section reads as though it previously said you could create multiple subscriptions in a single request by sending in an array of objects. That tells me this sentence could also be a strangely worded artefact because now we have this:

>  The POST request contains a single subscription object to be created.